doc/api/events.md

```diff
@@ -250,6 +264,8 @@ A class method that returns the number of listeners for the given `eventName`
 registered on the given `emitter`.
 
 ```js
+'use strict';
```

Comment:
This example both works without `use strict` and is partial (a continuation of the first example), that means that this is not the correct place for `use strict`.